utils_test.go

```diff
@@ -0,0 +1,41 @@
+package testcontainers_test
```

Comment:
I think this helper code and functions in it is outdated now that #2270 was merged

Reply:
Yep, I'll update now that this has merged 🙂 It all depended on which one merge first